tasks/2_convert-to-f/convert-to-f.js

```diff
@@ -0,0 +1,3 @@
+export function convertToF(temp) {
+  return temp*9/5 + 32;
```

Comment:
Should we be consistent here whether to separate math signs by spaces or not?
It should be `temp * 9 / 5 + 32` or `temp*9/5+32`.
Me personally like it better with separated.

Reply:
Changed.